pandas/tests/groupby/test_apply.py

```diff
@@ -901,3 +901,17 @@ def fn(x):
         name="col2",
     )
     tm.assert_series_equal(result, expected)
+
+
+def test_apply_reindex_values():
+    # GH: 26209
```

Comment:
a few words about what the bug was would be hepful

Reply:
done